packages/bot-web-ui/src/components/dashboard/dashboard-component/info-panel.tsx

```diff
@@ -20,47 +30,84 @@ const InfoPanel = ({ setInfoPanelVisibility, setActiveTab, setActiveTabTutorial
+
+                {SIDEBAR_INTRO.map((sidebar_item, index) => {
+                    const { label, content, link } = sidebar_item;
+                    return (
+                        <div key={`${label}-${index}`}>
                             <Text
-                                key={`info-panel-tour${key}`}
-                                className={classNames('db-info-panel__card', {
-                                    'db-info-panel__content': link,
-                                })}
                                 color='prominent'
-                                line_height='xl'
-                                as='p'
-                                onClick={() => switchTab(link, label)}
+                                line_height='xxl'
+                                size={is_mobile ? 's' : 'm'}
+                                weight='bold'
+                                as='h1'
                             >
-                                {text}
+                                {label}
                             </Text>
-                        ))}
+                            {content.map((text, key) => (
+                                <Text
+                                    key={`info-panel-tour${key}`}
+                                    className={classNames('db-info-panel__card', {
+                                        'db-info-panel__content': link,
+                                    })}
+                                    color='prominent'
+                                    line_height='xl'
+                                    as='p'
+                                    onClick={() => switchTab(link, label)}
+                                    size={is_mobile ? 'xxs' : 's'}
+                                >
+                                    {text}
+                                </Text>
+                            ))}
+                        </div>
+                    );
+                })}
+            </div>
+        );
+    };
+
+    return (
+        <>
+            <DesktopWrapper>
+                {!has_started_onboarding_tour && (
+                    <div
+                        className={classNames('tab__dashboard__info-panel', {
+                            'tab__dashboard__info-panel--active': is_info_panel_visible,
+                        })}
+                    >
+                        {renderInfo()}
                     </div>
-                );
-            })}
-        </div>
+                )}
+            </DesktopWrapper>
+            <MobileWrapper>
+                <Modal
+                    className={classNames('statistics__modal', { 'statistics__modal--mobile': is_mobile })}
+                    is_open={is_info_panel_visible}
+                    toggleModal={handleClose}
+                    width={'440px'}
+                >
+                    <Modal.Body>{renderInfo()}</Modal.Body>
+                </Modal>
+            </MobileWrapper>
```

Comment:
@sandeep-deriv  i think we don’t  need to check is **mobile** version or not inside `<MobileWrapper>`, because we check it inside of the component  
```suggestion
            <MobileWrapper>
                <Modal
                    className={classNames('statistics__modal', 'statistics__modal--mobile')}
                    is_open={is_info_panel_visible}
                    toggleModal={handleClose}
                    width={'440px'}
                >
                    <Modal.Body>{renderInfo()}</Modal.Body>
                </Modal>
            </MobileWrapper>
```

Reply:
yeah, you are right, let me correct it. ✅